scripts/portconfig

```diff
@@ -258,26 +263,30 @@ def main():
     # Load database config files
     load_db_config()
     try:
+        port_table_name = interface.get_port_table_name(args.port)
```

Comment:
have you tested this with device set to naming_mode as "alias" and if you passed the alias name instead would this still work?  I think we should move the naming mode check to here instead of handled at set_mtu only so that it should be handling the alias case in case naming_mode is set to alias?